dev/com.ibm.ws.jmx.connector.local/src/com/ibm/ws/jmx/connector/local/LocalConnectorActivator.java

```diff
@@ -70,7 +70,8 @@ public String run() {
                         String localConnectorAddress = null;
                         // Start the JMX agent and retrieve the local connector address.
                         if (JavaInfo.majorVersion() < 9 ||
-                            (JavaInfo.majorVersion() >= 9 && !Boolean.getBoolean("jdk.attach.allowAttachSelf"))) {
+                            (JavaInfo.majorVersion() >= 9 && (!Boolean.getBoolean("jdk.attach.allowAttachSelf") ||
```

Comment:
It may be good to include the value of com.ibm.tools.attach.enable in the debug statement below.  Also a comment explaining this change would possible be good for Java 9+